src/detect-engine-analyzer.c

```diff
@@ -861,6 +862,14 @@ static void DumpMatches(RuleAnalyzer *ctx, JsonBuilder *js, const SigMatchData *
                 jb_close(js);
                 break;
             }
+            case DETECT_ACK: {
+                const DetectAckData *cd = (const DetectAckData *)smd->ctx;
+
+                jb_open_object(js, "ack");
+                jb_set_uint(js, "number", cd->ack);
```

Comment:
@jasonish is this ok with the current work you are doing about rules parsing ?

Reply:
> @jasonish is this ok with the current work you are doing about rules parsing ?

Should be.  But I'd hope both be in 8, so not too worried.

Reply:
Could you elaborate a bit what your concern is here, @catenacyber ?

Reply:
No concern if Jason is ok :-)